File: PooCC01CNetz/Semana03/Python/Ejercicios2/MiNegocioLibros.py
from Libro import *
class MiNegocioLibros:
    def main():
        lib = Libro()
        lib.setIsbn("ABC123HDF")
        lib.setTitulo("Calculo Diferencial")
        lib.setAutor("Netz Romero")
        # Visualiza los atributos con el metodo imprimirAtributos
        lib.imprimirAtributos()
        # Visualiza los atributos con los getters
        print("ISBN    : " + lib.getIsbn())
        print("Titulo  : " + lib.getTitulo())
        print("Autor   : " + lib.getAutor())
    
    main()

File: PooCC01CNetz/Semana03/Java/Ejercicios2/MiNegocioLibros.java
public class MiNegocioLibros {
    public static void main(String[] args) {
        Libro lib = new Libro();
        lib.setIsbn("ABC123HDF");
        lib.setTitulo("Calculo Diferencial");
        lib.setAutor("Netz Romero");
        // Visualiza los atributos con el metodo imprimirAtributos
        lib.imprimirAtributos();
        // Visualiza los atributos con los getters
        System.out.println("ISBN    : " + lib.getIsbn());
        System.out.println("Titulo  : " + lib.getTitulo());
        System.out.println("Autor   : " + lib.getAutor());
    }
}